### ISSUE TITLE
Relayer URL change 

### ISSUE THREAD

**Issue by UmckaBear** [BuildOnViction/bug-reports#113]
By default URL field for relayer is empty. You can add URL, but when you try to remove it, page became blank without saving changes. If url contains ":", then it automatically falls under the definition of the correct link, even it contains 2 characters.
Examples:
"w:" - correct for check
"wwww" - doesn't correct

Video example below:
https://drive.google.com/file/d/1J9-FQopXwlAqiAtytx7nDRzLEpxEZ6ec/view?usp=sharing

Also below examples how URL looks like on Relayer Page
![URLExamples](https://user-images.githubusercontent.com/31537952/73487957-a634ba80-43b0-11ea-8419-c10721f8e8f0.jpg)

It should allow apply empty URL value, or there should be warning, that URL can't be blank, but not page crash.
There should be correct URL check. The correctness of the link should not be tied to just one character ":". It can be http://name, or https://name, or http://IP, or https://IP, or no check for symbols. Warning message should indicate the possible template.

**Comment by thanhson1085** [comment]
Yep, we should validate the inputs, that i created the issue https://github.com/tomochain/tomorelayer/issues/73
